packages/kbn-cli-dev-mode/src/base_path_proxy_server.ts

```diff
@@ -21,8 +21,7 @@ import { DevConfig, HttpConfig } from './config';
 import { Log } from './log';
 
 const ONE_GIGABYTE = 1024 * 1024 * 1024;
-const alphabet = 'abcdefghijklmnopqrztuvwxyz'.split('');
-const getRandomBasePath = () => sampleSize(alphabet, 3).join('');
```

Comment:
my initial implementation to retry `sampleSize` until the result was `tim` was a bit flaky